frontend/lib/src/components/widgets/NumberInput/NumberInput.tsx

```diff
@@ -410,30 +410,32 @@ class NumberInput extends React.PureComponent<Props, State> {
               },
             }}
           />
-          <StyledInputControls>
-            <StyledInputControl
-              className="step-down"
-              onClick={this.modifyValueUsingStep("decrement")}
-              disabled={disableDecrement}
-            >
-              <Icon
-                content={Minus}
-                size="xs"
-                color={this.canDecrement ? "inherit" : "disabled"}
-              />
-            </StyledInputControl>
-            <StyledInputControl
-              className="step-up"
-              onClick={this.modifyValueUsingStep("increment")}
-              disabled={disableIncrement}
-            >
-              <Icon
-                content={Plus}
-                size="xs"
-                color={this.canIncrement ? "inherit" : "disabled"}
-              />
-            </StyledInputControl>
-          </StyledInputControls>
+          {width > 120 && (
```

Comment:
Is there a reasonable place to put this breakpoint in theme where it is easily accessible here? Maybe `frontend/lib/src/theme/primitives/breakpoints.ts`? Also please add a note there as to what its used for, perhaps why 120px.

Also think we should make this a variable above the render with explanatory comment above it, something along the lines of:
```
// We only want to show the increment/decrement controls when there is sufficient room
// to display the value & these controls
const showControls = theme.breakpoints.numberInput > 120
```

Reply:
Sorry for the poor job at explaining how this works: Updated in [95b0381](https://github.com/streamlit/streamlit/pull/7050/commits/95b038195ea6e4b474252fa5dc83bd102bcd26b1)